File: launchdarkly/ruby/src/GetContextsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::ContextsApi.new.get_contexts(
        "projectKey_string", # project_key
        "environmentKey_string", # environment_key
        "kind_string", # kind
        "key_string", # key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling ContextsApi#get_contexts: #{e}"
end

File: launchdarkly/typescript-axios/src/GetContextsExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ContextsApi(configuration).getContexts(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "kind_string", // kind
  "key_string", // key
  undefined, // limit
  undefined, // continuationToken
  undefined, // sort
  undefined, // filter
  undefined, // includeTotalCount
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ContextsApi#getContexts:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
